spec/lib/submission_builder/ty2022/states/ny/individual_return_spec.rb

```diff
@@ -34,5 +34,18 @@
         expect(xml.at("IT215 E_FED_EITC_IND").attribute('claimed').value).to eq("1")
       end
     end
+
+    context "when getting a refund to a personal checking account" do
+      let(:intake) { create(:state_file_ny_intake, filing_status: filing_status, routing_number: "011234567", account_number: "123456789", account_type: 1) }
+      let(:filing_status) { 'single' }
+
+      it 'generates XML from the database models' do
+        xml = Nokogiri::XML::Document.parse(described_class.build(submission).document.to_xml)
```

Comment:
for what it's worth `described_class.build(submission).document` is already a Nokogiri::XML::Document so we shouldn't need to `.to_xml` it and parse again

but it looks like this pattern is in use plenty of places and I copied it into this file myself so 🤷

Reply:
fixed all the instances in this file